webapp/components/wpt-results.html

```diff
@@ -431,6 +431,11 @@ <h3>
         const nextSubdirIndex = this.path === '/' ? 0 : currentPathParts.length;
 
         let displayedTests = [];
+        if (this.computePathIsATestFile(this.path)) {
+          this.displayedTests.push(this.path);
```

Comment:
This seems wrong. It should be `this.displayedTests = [this.path]`.

A quick debugging confirms my guess. `this.displayedTests` contains two duplicate test files when we're browsing single-test results (the function is called twice), although nothing the UI doesn't show any issue.